UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Now that I'm on my Mac, here's the screen capture of that setting in Xcode that make Xcode create empty lines unintended
<img width="862" alt="xcode-setting-newlines" src="https://cloud.githubusercontent.com/assets/216089/15807650/ab57000e-2b63-11e6-935c-43fac5398d09.png">

_Note that you can fix all those warnings automatically using some `swiftlint autocorrect` command in the Terminal instead of fixing them one by one manually_

---

Also, I just added a `as!` in a template just to check, and confirm that SwiftGen did generate a warning for it, so not sure why you don't have it:
<img width="888" alt="Force-Cast SwiftLint error" src="https://cloud.githubusercontent.com/assets/216089/15807638/5e823b72-2b63-11e6-9095-15dcbb87a7f0.png">
I realize by reading my script again though that it does scan for `*.swift.out` files only at the root level of the `UnitTests/expected` folder, but not in subfolders if any. I'm gonna fix that in an upcoming commit.